ui/src/Pages/Preferences/Invites/ManageInvites.jsx

```diff
@@ -34,18 +34,41 @@ function ManageInvites() {
     [dispatch]
   );
 
+  const toClipboard = (token) => {
+    navigator.clipboard.writeText(token).then(
+      () => {
+        dispatch(
+          addNotification({
+            msg: "Successfuly copied to your clipboard.",
+          })
+        );
+      },
+      () => {
+        dispatch(
+          addNotification({
+            msg: "There was an error copying your id to your clipboard",
+          })
+        );
+      }
+    );
+  };
+
   const tokens = auth.invites.items.map((token, i) => {
     const { hours, mins, secs, date, month, year } = formatHHMMSSDate(
       token.created
     );
 
     return (
-      <div className="token" key={i}>
+      <div className="token" key={i} onClick={() => toClipboard(token.id)}>
```

Comment:
At the moment when hovering over the id button, the cursor is either a pointer or a text editor cursor. We should add some css that changes the cursor to "clickable" when hovering over the token,